colossus-docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,25 @@
+@@@ index
```

Comment:
There is an io-overview.png and iosystem.png images that aren't used. Should they? Or should I delete?